src/main/java/carracing/controller/CarRacingGameController.java

```diff
@@ -0,0 +1,39 @@
+package carracing.controller;
+
+import carracing.domain.*;
+import carracing.util.RandomGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 자동차 경주 게임을 구현한 클래스
+ */
+public class CarRacingGameController {
+    private final RandomGenerator randomGenerator = new RandomGenerator();
+    private final MovementPolicy movementPolicy = new RandomIntMovementPolicy(randomGenerator);
```

Comment:
```suggestion
    private final MovementPolicy movementPolicy = new RandomIntMovementPolicy(new RandomGenerator());
```
개발 성향 차이라고 느끼지만 다른 곳에서 사용하지 않는다면 변수로 할당하지 않아도 괜찮을 것 같아요

Reply:
네 말씀이 맞습니다. 변수로 객체를 생성할 필요는 없겠네요. 👍